apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -557,12 +557,21 @@ export async function linkAssessment(
   });
 }
 
-/* Throttling notes
-// https://canvas.instructure.com/doc/api/file.throttling.html
-// 403 Forbidden (Rate Limit Exceeded)
-// X-Request-Cost
-// X-Rate-Limit-Remaining
-*/
+function findValueByKey(obj: unknown, targetKey: string) {
```

Comment:
This could benefit from a JSDoc comment, and it'd probably be pretty easy to add a few unit tests to in an adjacent `lti13.test.ts` file.